examples/example0005-accelerate_backend.lhs

```diff
@@ -0,0 +1,42 @@
+
+> {-# LANGUAGE NoImplicitPrelude #-}
+> {-# LANGUAGE RebindableSyntax #-}
+> {-# LANGUAGE OverloadedLists #-}
+> {-# LANGUAGE TypeOperators #-}
+> {-# LANGUAGE FlexibleContexts #-}
+> {-# LANGUAGE GADTs #-}
+> {-# LANGUAGE DataKinds #-}
+> import qualified Prelude as P
+> import SubHask
+> import SubHask.Algebra.Vector (ACCVector, UVector, mkAccVector, mkAccVectorFromList)
+> import SubHask.Algebra.Matrix (ACCMatrix, Matrix, unsafeToModuleM, mkAccMatrixFromList, mkAccMatrixFromMatrix)
+> import SubHask.Algebra.Accelerate (ValidBackend(..))
+> import SubHask.Algebra.AccelerateBackend (Backend(..))
+> import System.IO
+>
+> v  :: ACCVector Interpreter "a" Double
+> v  = mkAccVectorFromList [0..5]
+>
+> v' :: ACCVector Interpreter "a" Double
+> v' = mkAccVectorFromList [0..5]
+>
+>
+> mmat  :: Matrix (UVector "v" Double) Double "a" "b"
+> mmat  = unsafeToModuleM 2 [0..5]
+>
+> m :: ACCMatrix Interpreter (ACCVector Interpreter "v" Double ) "a" "b" Double
+> m = mkAccMatrixFromMatrix mmat
+>
+> mm :: ACCMatrix Interpreter (ACCVector Interpreter "v" Double ) "a" "b" Double
```

Comment:
Have you tested on the `CUDA` or `LLVM` backends yet, or just the `Interpreter`?